packages/frameworks-sveltekit/src/lib/index.ts

```diff
@@ -302,6 +331,8 @@ declare global {
   namespace App {
     interface Locals {
       getSession(): Promise<Session | null>
+      getSessionWithSetCookies(): Promise<{ session: Session, cookies: Record<string, Record<string, string>> } | null>
```

Comment:
I wonder if we should modify the `getSession` function's return instead 🤔 since we're still at 0.x.x versioning.

Reply:
Yeah tbh I think thats fine too, will be cleaner in the end, right

Reply:
Removed the new method and made the new behavior behind `getSession` :+1: